docs/jsonrpc_apis.md

```diff
@@ -607,34 +597,33 @@ results:
-| Version     | integer         | node's version in config.json                               |
-| Services    | integer         | node service type. 4 is spv service and 0 is no spv service |
-| Relay       | bool            | whether node will relay transaction or not                  |
-| TxnCnt      | integer         | transactions transmitted by this node                       |
-| RxTxnCnt    | integer         | The transaction received by this node                       |
-| Port        | integer         | p2p network port                                            |
-| RPCPort     | integer         | json-RPC service port                                       |
+| Height      | integer         | current height of local node                                |
+| Version     | integer         | peer-to-peer network protocol version of this node          |
+| Services    | string          | the services provided by this node                          |
+| Port        | integer         | P2P network port                                            |
+| RPCPort     | integer         | JSON-RPC service port                                       |
 | RestPort    | integer         | RESTful service port                                        |
 | WSPort      | integer         | webservice port                                             |
-| OpenPort    | integer         | open service port                                           |
-| OpenService | bool            | if opens service enabled                                    |
 | Neighbors   | array[neighbor] | neighbor nodes information                                  |
 
 neighbor:
 
-| name       | type    | description                                                     |
-| ---------- | ------- | --------------------------------------------------------------- |
-| ID         | integer | neighbor's id                                                   |
-| HexID      | string  | neighbor's id in hex format                                     |
-| Height     | integer | neighbor current height                                         |
-| Services   | integer | neighbor service type. 4 is spv service and 0 is no spv service |
-| Relay      | bool    | whether neighbor will relay transaction or not                  |
-| External   | bool    | whether neighbor is from external network                       |
-| State      | string  | neighbor state in string format                                 |
-| NetAddress | string  | neighbor tcp address                                            |
+| name           | type    | description                                                     |
+| -------------- | ------- | --------------------------------------------------------------- |
+| ID             | integer | a random number to identify the neighbor instance               |
+| Addr           | string  | network address of the neighbor in host:port format             |
```

Comment:
Full and lowercase should be used in the interface

Reply:
Fixed.